app/controllers/conferences_controller.rb

```diff
@@ -2,17 +2,24 @@ class ConferencesController < ApplicationController
   protect_from_forgery with: :null_session
   before_action :respond_to_options
   load_and_authorize_resource find_by: :short_title
-  load_resource :program, through: :conference, singleton: true, except: :index
 
   def index
     @current = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)
     @antiquated = @conferences - @current
   end
 
-  def show; end
+  def show
+    # have to change "localhost" to ENV['OSEM_HOSTNAME'] in production
+    check_custom_domain if request.host != 'localhost'
```

Comment:
Also: Maybe we can call `check_custom_domain` ONLY if `params[:short_title]` is not set?
That way, we don't need to do that `request.host != 'localhost'` check (some people might use 0.0.0.0:3000 for example).